android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -91,40 +88,6 @@ class Screen constructor(context: ReactContext?) : ViewGroup(context), FabricVie
             })
```

Comment:
Maybe let's change the function to `updateScreenSize`?
And then on paper add this implementation and on fabric the fabric version?